FluidNC/src/Protocol.cpp

```diff
@@ -828,11 +828,17 @@ void protocol_exec_rt_system() {
     if (rtSafetyDoor) {
         protocol_do_safety_door();
     }
-
-    Maslow.recomputePID(); //This one works as an alternative to having recomputePID called in DCServo.cpp
+  
+    //Maslow.recomputePID(); //This one works as an alternative to having recomputePID called in DCServo.cpp
 
     protocol_handle_events();
 
+    
+    //do all the Maslow stuff here
+    if(!Maslow.using_default_config && ( sys.state() == State::Jog || sys.state() == State::Cycle ) ){
```

Comment:
How does this work? Why do we only do it when sys.state == jog or cycle? Not complaining, just want to understand

Reply:
I _think_, those are the only two states where the FluidNC issues it commands to move. I was thinking about adding the Hold there too, but it seems to hold itself in place fine passively?

Reply:
Are we still updating the encoders the rest of the time? I'm pretty sure we need to do that all the time to make sure that they can't move when it's not looking

Reply:
Like we're probably not updating them?